src/scss/components/navigation/_index.scss

```diff
@@ -93,3 +100,37 @@
     }
   }//hx-reveal
 }
+
+// ===== SUB-LEFT NAVIGATION =====
+.sub-leftNav, .first-section {
```

Comment:
Please modify to add a `hx` prefix to any Helix classes we add.

### Please change:

```scss
.sub-leftNav
```

### To:

```scss
.hxLeftNavSection
```

Reply:
### Please change:

```css
.first-section
```

To:

```css
.hxLeftNavSubSection
```

Reply:
Changes have been done, Please review